app/views/shared/_supporting_information.html.erb

```diff
@@ -81,12 +67,14 @@
     <div id="accordion-default-content-3" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-3">
       <% if @agent %>
         <p class='govuk-body'>
-          <strong>Agent: </strong><br><%= h(full_details(@agent)) %>
+          <strong>Agent: </strong><br>
+          <%= h(full_details(@agent)) %>
         </p>
-        <% end %>
+      <% end %>
       <% if @applicant %>
         <p class='govuk-body'>
-          <strong>Applicant: </strong><br><%= h(full_details(@applicant)) %>
+          <strong>Applicant: </strong><br>
+          <%= h(full_details(@applicant)) %>
```

Comment:
note: these line split modifications were made automatically by htmlbeautifier, I can put it back if this is a problem.

Reply:
Looks like an improvement to me.